From bf04560d54c5afe6f059e66892ee560971862de8 Mon Sep 17 00:00:00 2001
From: Venkateshwar Rao G <venkateshwar.rao.gannavarapu@xilinx.com>
Date: Wed, 14 Mar 2018 19:53:19 +0530
Subject: [PATCH 0331/1851] dt: bindings: display: xlnx: Updated gpio reset
 entry

commit 0c0bca33fee739e93081bcc66f65dae213746c6e from
https://github.com/Xilinx/linux-xlnx.git

This patch updates reset gpio entry arguments.

Signed-off-by: Venkateshwar Rao G <venkateshwar.rao.gannavarapu@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/display/xlnx/xlnx,vpss-scaler.txt     | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Documentation/devicetree/bindings/display/xlnx/xlnx,vpss-scaler.txt b/Documentation/devicetree/bindings/display/xlnx/xlnx,vpss-scaler.txt
index 9d46dc28807b..f3e30e7daa75 100644
--- a/Documentation/devicetree/bindings/display/xlnx/xlnx,vpss-scaler.txt
+++ b/Documentation/devicetree/bindings/display/xlnx/xlnx,vpss-scaler.txt
@@ -22,13 +22,13 @@ Required properties:
 - xlnx,pix-per-clk : The pixels per clock property of the IP.
   supported values are 1 and 2.
 
-- reset-gpios: Specifier for a GPIO that assert for VPSS Scaler reset.
+- reset-gpios: GPIO specifier to assert/de-assert the reset line.
 
 Example:
 	scaler@a0040000 {
 		compatible = "xlnx,vpss-scaler";
 		reg = <0x0 0xa0000000 0x0 0x40000>;
-		reset-gpios = <&axi_gpio_1 0x0 0x0>;
+		reset-gpios = <&gpio 0x0 GPIO_ACTIVE_LOW>;
 		xlnx,num-hori-taps = <8>;
 		xlnx,num-vert-taps = <8>;
 		xlnx,pix-per-clk = <2>;
-- 
2.31.1

